lib/compat/wordpress-6.6/compat.php

```diff
@@ -15,11 +15,14 @@
 function gutenberg_change_patterns_link_and_remove_template_parts_submenu_item() {
 	if ( ! wp_is_block_theme() ) {
 		global $submenu;
-		foreach ( $submenu['themes.php'] as $key => $item ) {
-			if ( 'edit.php?post_type=wp_block' === $item[2] ) {
-				$submenu['themes.php'][ $key ][2] = 'site-editor.php?path=/patterns';
-			} elseif ( 'site-editor.php?path=/wp_template_part/all' === $item[2] ) {
-				unset( $submenu['themes.php'][ $key ] );
+
+		if ( ! empty( $submenu['themes.php'] ) ) {
```

Comment:
Let's use early return and reduce the nesting level.

Reply:
Updated 👍